projects/vdk-core/cicd/simple-functional-test.sh

```diff
@@ -54,19 +54,19 @@ python setup.py sdist --formats=gztar && pip install dist/* || fail "VDK Install
 
 function restore_vdk_in_editable_mode {
   pip install -e .
-  pip install --extra-index-url $PIP_EXTRA_INDEX_URL -e ./plugins/vdk-trino
-  pip install --extra-index-url $PIP_EXTRA_INDEX_URL -e ./plugins/vdk-plugin-control-cli
+  pip install --extra-index-url $PIP_EXTRA_INDEX_URL -e ./../vdk-plugins/vdk-trino
+  pip install --extra-index-url $PIP_EXTRA_INDEX_URL -e ./../vdk-plugins/vdk-plugin-control-cli
 }
 trap restore_vdk_in_editable_mode EXIT
 
 echo "Install database plugin for presto support"
-pushd plugins/vdk-trino || exit
+pushd ../vdk-plugins/vdk-trino || exit
```

Comment:
Let's not change it here in this PR as this is way too big change.

 But I want each high-level project (https://github.com/vmware/versatile-data-kit/tree/main/projects) to be independent of each other (as if they are in separate git repositories). 
 
 So we'd need to see how to fix that so we do not have cross project references (../vdk-plugins). But definitely in separate PR